### ISSUE TITLE
Script to clear background task backlog

### ISSUE THREAD

**Issue by Steven-Eardley** [DOAJ/doaj#1471]
If the connection to redis fails but jobs are requested, we see the following message in the logs:
```
QueueWriteException: ConnectionError: Error 111 connecting to 127.0.0.1:6379. Connection refused.
```

The task is still saved to ES, but will remain in queued state forever, because it's not actually on the queue. We need a script to clear this backlog of tasks by re-pushing them on to the queue, oldest to newest.

**Comment by Steven-Eardley** [comment]
It may be useful to supply a date range to this, so we don't re-enqueue tasks that are newer than the zombie tasks.

**Comment by Steven-Eardley** [comment]
Completed, merged in #1472
